testing/single/src/docstrings.py

```diff
@@ -0,0 +1,1295 @@
+# This file is for testing different types of docstrings and verifying they display
+# correctly.
+# Try it with `python.analysis.supportRestructuredText` both true and false. 
+import numpy as np
+
+# Hover over zeroes and ensure the markdown looks correct.
+z = np.zeros([1, 2, 3])
+
+# Hover over take and ensure the markdown looks correct.
```

Comment:
Can you be more specific about what looks correct or rendered correctly means here so that we know what to look for?

Reply:
I can't. That's the whole problem. It's subjective. I'll say that.